lib/igniter.ex

```diff
@@ -1417,7 +1431,8 @@ defmodule Igniter do
         warnings: Enum.uniq(igniter.warnings),
         tasks: Enum.uniq(igniter.tasks)
     }
-    |> Igniter.Project.Module.move_files()
+    # FIXME
+    # |> Igniter.Project.Module.move_files()
```

Comment:
I noticed 2 issues with `move_files/0`

1. Rules are incomplete and it ends up placing files in the wrong place, for eg: `my_app_web/core_components.ex` instead of `my_app_web/components/core_components.ex`

2. This causes a mismatch with tests because it only runs in the actual execution of the pipeline, ie: in `test_project()` it will show `my_app_web/components/core_components.ex` but that's not true.

Reply:
So two things from this:

1. does that happen even with the phoenix plugin added? If so, then we need to adjust the plugin.
2. we need to add the move_files step to the test project code when we apply the igniter in tests in that case. I think the file moving logic is pretty important, it prevents igniters from adding files to places that don't match the users preferred file location pattern (combined with plugins). With that said, you can create a file *and* update the `dont_move_files` config in the `igniter.exs` at the same time.

Reply:
Adding `Igniter.compose_task(igniter, "igniter.add_extension", ["phoenix"])` seems to have fixed the issue 👍🏻